src/cljs/org/broadinstitute/firecloud_ui/page/workspace/workspace_method_confs.cljs

```diff
@@ -36,10 +38,126 @@
+                       :cell-renderer (fn [row-num conf] (cell (str ((conf "methodStoreMethod") "methodNamespace") ":"
+                                                                 ((conf "methodStoreMethod") "methodName") ":"
+                                                                 ((conf "methodStoreMethod") "methodVersion"))))}
+                      {:header-component (header "Config") :starting-width 290
+                       :cell-renderer (fn [row-num conf] (cell (str ((conf "methodStoreConfig") "methodConfigNamespace") ":"
+                                                                 ((conf "methodStoreConfig") "methodConfigName") ":"
+                                                                 ((conf "methodStoreConfig") "methodConfigVersion"))))}
+                      {:header-component (header "Inputs") :starting-width 200
+                       :cell-renderer (fn [row-num conf] (cell (stringify_map (conf "inputs"))))}
+                      {:header-component (header "Outputs") :starting-width 200
+                       :cell-renderer (fn [row-num conf] (cell (stringify_map (conf "outputs"))))}
+                      {:header-component (header "Prerequisites") :starting-width 200
+                       :cell-renderer (fn [row-num conf] (cell (stringify_map (conf "prerequisites"))))}]
+            :data (:method-confs props)
+            :row-props (fn [row-num conf]
+                         {:style {:fontSize "80%" :fontWeight 500
+                                  :paddingTop 10 :paddingBottom 7
+                                  :backgroundColor (if (even? row-num) (:background-gray style/colors) "#fff")}})})])])})
+
+
+(defn- modal-import-background [state]
+  {:backgroundColor "rgba(82, 129, 197, 0.4)"
+   :display (when-not (:import-overlay-shown? @state) "none")
+   :overflowX "hidden" :overflowY "scroll"
+   :position "fixed" :zIndex 9999
+   :top 0 :right 0 :bottom 0 :left 0})
+
+(def ^:private modal-import-content
+  {
+   ;:transform "translate(-50%, 0px)" ORIGINAL
+   :transform "translate(-50%,0px)"
+   :backgroundColor (:background-gray style/colors)
+   :position "relative" :marginBottom 60
+   :top 60
+   ;:left "50%"  ORIGINAL
+   :left "50%"
+   ;:width 500 ORIGINAL
+   :width "75%"
+   })
+
+
+(defn- render-import-overlay [state refs]
+  (let [clear-import-overlay (fn []
+      (common/clear! refs "wsName" "wsDesc" "shareWith")
+      (swap! state assoc :import-overlay-shown? false))]
+    [:div {
+      :style (modal-import-background state)
+      :onKeyDown (common/create-key-handler [:esc] clear-import-overlay)}
+    [:div {:style modal-import-content}
+      [:div {:style {:backgroundColor "#fff"
+                     :borderBottom (str "1px solid " (:line-gray style/colors))
+                     :padding "20px 48px 18px"
+                     }}
+      [:div {}
+        [:div {:style {:paddingLeft "100%"    :halign "right"  :align "right"}} " "
```

Comment:
I was thinking we should probably make this button's position absolute, and place it directly on the corner of the dialog.  If you don't want to mess with that I can try to do it in a follow-up.